src/main/java/com/googleinterns/zoomtube/data/Comment.java

```diff
@@ -49,6 +49,11 @@
    */
   public abstract Optional<Long> timestampMs();
 
+  /**
+   * Returns the key of the transcript line where an indicator should appear.
```

Comment:
So for encapsulation the Comment knows nothing about the TranscriptLine indicator.  I would rephrase this as:

Returns the transcript line key for which this comment is for.

Reply:
done!